pkg/client/unversioned/remotecommand/remotecommand.go

```diff
@@ -78,7 +80,8 @@ func NewExecutor(config *restclient.Config, method string, url *url.URL) (Stream
 		return nil, err
 	}
 
-	upgradeRoundTripper := spdy.NewRoundTripper(tlsConfig)
+	followRedirects := utilfeature.DefaultFeatureGate.Enabled(genericfeatures.StreamingProxyRedirects)
```

Comment:
This isn't actually used in client code, is it? Maybe I'm confused about how this is used.